### DISCUSSION TITLE
Add Features section to transifex-cli readme

### DISCUSSION THREAD

**Discussion by vexx32** [chocolatey-community/chocolatey-packages#2167] [Ideas]
RE: https://github.com/chocolatey-community/chocolatey-packages/pull/2164

The [Transifex docs](https://developers.transifex.com/docs/cli) for this new CLI are purely usage-based so they don't really do a great job of giving a summary of the features. We'll need to at some point go through and try to get a shortlist of the features to include in the package readme.

**Comment by AdmiringWorm** [comment]
To give a bit more context around this, we usually would require a `Features` section in the description of the package, however since (as mentioned) these features aren't easily extracted at the moment for the upstream software, and there is a time-sensitive nature to the package as the original `transifex-client` package was deprecated from the developers in favor of this new software we will be merging the changes without having this section available.

Once we have merged this PR, an issue will be created based on any discussions made here, even if no discussions happen.

I encourage community members to give their opinion in this discussion if they have any.

**Comment by pauby** [comment]
It looks like even the [previous package](https://community.chocolatey.org/packages/transifex-client) didn't have a list of features.

**Reply by AdmiringWorm** [reply]
I am aware, but it should have had one.
All packages should have a relevant features section even if the upstream software doesn't specify them.
